src/browser/modules/DBMSInfo/DatabaseSelector.tsx

```diff
@@ -70,12 +70,16 @@ export const DatabaseSelector = ({
           onChange={selectionChange}
         >
           {uniqDatabases.map(db => {
-            const defaultStr = db.default ? ' - default' : ''
+            const defaultOrHomeStr = db.home
+              ? ' - home'
+              : db.default
+              ? ' - default'
+              : ''
```

Comment:
I'm generally not a big fan of nested ternaries, but I think this is readable enough since it's simple

Reply:
yeah, I don't like massive nested ternaries either, but as you said, this case is simple & concise enough to be ok imo.